pkg/chat/handler.go

```diff
@@ -0,0 +1,126 @@
+package chat
+
+import (
+	"github.com/AsterNighT/software-engineering-backend/api"
+	"github.com/labstack/echo/v4"
+)
+
+type ChatHandler struct {
+}
+
+// @Summary New a chat
+// @Description
+// @Tags Chat
+// @Produce json
+// @Param chatDetail body Chat true "patient ID, doctor ID and other chat details"
+// @Success 200 {object} api.ReturnedData{data=Chat}
```

Comment:
```suggestion
// @Success 200 {object} api.ReturnedData{}
```

Reply:
I don't understand it...Why do we need to return data(in Message form) when a message is newed,but we don't need to return data (in Chat form) when a chat is newed?